include/lug/Window/Event.hpp

```diff
@@ -1,17 +1,55 @@
 #pragma once
 
 #include <lug/Window/Export.hpp>
+#include <lug/Window/Keyboard.hpp>
+
+#include <cstdint>
 
 namespace lug {
 namespace Window {
 
+/**
+ * Type of event
+ */
 enum class LUG_WINDOW_API EventType : uint32_t {
```

Comment:
We can maybe change that and put in the `Event` structure. And after that use it with `Event::Type`. Just to preserve the coherency of the code.